frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx

```diff
@@ -307,7 +310,11 @@ const wizardDispatchToProps = (dispatch, props) => ({
         data: {
           isCreateTemplate: props.isCreateTemplate,
           isProviderImport: props.isProviderImport,
-          userTemplateName: props.userTemplateName,
+          userTemplateName: {
+            value: props.userTemplateName,
+            initialized: false,
+            isValid: true,
```

Comment:
we should not load all the userTemplate, but just the one we need.

In that case we don't need isValid value here, because the validity is already present in FirehoseResult object


It is a fluid value. It could easily happen that we open wizard and then somebody deletes the userTemplate - and in that case the wizard stops being valid